test/test_default_dict.jl

```diff
@@ -92,6 +96,33 @@ end
     @test e['e'] == 9
     @test f['e'] == 0
 
+    # passkey
```

Comment:
this is a good time to use a nexted testset, rather than a comment delimited block.
We don't use them enough in DataStructures.jl.
I assume because they didn't used to exist.

Arguably, it would be weird to start now.
And better to go and clean everything at once.
But idk that that is a good argument.